be/src/exec/pipeline/sort/sort_context.h

```diff
@@ -232,6 +234,26 @@ class SortContext final : public ContextWithDependency {
 
     size_t _next_output_row = 0;
 };
+class SortContextFactory;
+using SortContextFactoryPtr = std::shared_ptr<SortContextFactory>;
+class SortContextFactory {
+public:
+    SortContextFactory(bool is_merging, int64_t limit, int32_t num_right_sinkers,
```

Comment:
Why sort context need a Factory? which seems confusing.

Reply:
Sort context is shared by PartitionSortSinkOperator and LocalMergeSortSourceOperator.